pkg/reconciler/ingress/resources/virtual_service_test.go

```diff
@@ -465,7 +465,9 @@ func TestMakeMeshVirtualServiceSpec_CorrectRetries(t *testing.T) {
 					},
 				}}},
 		},
-		expected: nil,
+		expected: &istiov1alpha3.HTTPRetry{
+			Attempts: 0,
+		},
```

Comment:
```suggestion
		expected: &istiov1alpha3.HTTPRetry{},
```
Should work. 
Same below